ansible-vault/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "ansible-vault" %}
+{% set version = "2.0.0b1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "e52592058143b4e5e66bba234b4cba8056c39967c0aac5950743c3f985f0e054"
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - ansible
```

Comment:
is ansible here really needed?

Reply:
That's right, thanks @bgruening , I've removed this dependency.
 - Also moved `meta.yaml` into its correct place
 
The linter notes that
> License is not an SPDX identifier (or a custom LicenseRef) nor an SPDX license expression.
I don't know how to fix this, if you tell me I'm happy to update the license.

Reply:
@bgruening: without the `host: ansible` requirement, the windows build is failing:
https://ci.appveyor.com/project/conda-forge/staged-recipes/builds/31788222

This happens because when calling `import ansible_vault`, `ansible_vault` wants to import from `ansible` right away. 

So if I understand correctly, `ansible` is not required for the build, but it is required to run the `import ansible_vault` test. And the _Creating Conda Recipes_ doc says to make sure the module is imported as a bare minimum for testing, so we can:
- Keep `host:ansible` 
- Remove the test

What's the best way to resolve this?

Reply:
You can add `ansible` as a test requirement

Reply:
@mariusvniekerk I some your guidance here.
I added `ansible` as a test requirement, but the tests fails (ansible not found).

Also, `ansible` has been listed as a `run` requirement since the original PR, and if I understand the doc correctly it should be added automatically as a test requirement:
> In addition to the runtime requirements, you can specify requirements needed during testing. The runtime requirements that you specified in the "run" section described above are automatically included during testing.

Still no luck with importing `ansible` while testing, only if I explicitly add it as a `host` requirement.

Reply:
Forget about it for now, this won't be a `meta.yaml` issue, I'm working on it